Sources/BraintreeThreeDSecure/BTThreeDSecureClient.swift

```diff
@@ -424,9 +424,9 @@ import BraintreeCore
             requestParameters["customFields"] = customFields
         }
 
-        if request._cardAddChallenge == .requested || request.cardAddChallengeRequested == true {
+        if request.cardAddChallengeRequested == true {
             requestParameters["cardAdd"] = true
-        } else if request._cardAddChallenge == .notRequested {
+        } else if request.cardAddChallengeRequested == false {
```

Comment:
question for the team: does this logic sound okay? (updated this after removing the `cardAddChallenge` property based on its doc strings)

Reply:
There is a separate ticket for this: DTMOBILES-1140. So we can only do the init portion in this PR.

Reply:
ahh gotcha! thanks, addressed in b2bec4a7a7638818e90bb3b13d658912b0b9a9a4

Reply:
I think we can revert the changes here and uncomment `_cardAddChallenge` for now, then we can tackle all of it in a single PR.